README.md

```diff
@@ -14,6 +14,8 @@
 <img src="https://img.shields.io/github/stars/middlewarehq/middleware?style=for-the-badge" alt="Stars" />
 </p>
 
+<!-- [Join our Engineering Leaders Community]() -->
```

Comment:
@aravindputrevu did you intend to leave this commented out for now and get this merged? OR should we wait for the link to be here?

Reply:
Next PR once you approve 🙂